src/features/flows/controllingConfig.ts

```diff
@@ -0,0 +1,232 @@
+        controllingGuide: [
+            {
+                name: 'Apple Home',
+                guide: [
+                    'Open the <b>Apple Home</b> app',
+                    'Navigate to the room that contains the device that was previously paired',
+                    'Click on the light bulb symbol on the <b>Matter Accessory</b> tile to turn on the Matter Light Bulb from the room view',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns off</b>',
+                    'Click on the light bulb symbol on the <b>Matter Accessory</b> tile again to turn off the Matter Light Bulb from the room view',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns on</b>',
+                    'Tap the <b>Matter Accessory</b> Light Bulb tile to open detailed view',
+                    'Tap on the slider to change the brightness of the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>changed brightness level</b> accordingly',
+                ],
+                video: '../resources/ecosystems/Apple/usage/apple_bulb_usage.mp4',
+            },
+            {
+                name: 'SmartThings',
+                guide: [
+                    'Open the <b>SmartThings</b> app',
+                    'Go to the <b>Devices</b> page',
+                    'Tap the power symbol on the <b>MatterLight 1</b> Light Bulb tile to to turn on the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns on</b>',
+                    'Tap on the <b>MatterLight 1</b> Light Bulb tile to open detailed view',
+                    'Click on the power symbol again to turn off the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns off</b>',
+                    'Move the slider to change the brightness of the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>changed brightness level</b> accordingly',
+                ],
+                video: '../resources/ecosystems/SmartThings/usage/smartthings_bulb_usage.mp4',
+            },
+            {
+                name: 'Amazon Alexa',
+                guide: [
+                    'Open the <b>Amazon Alexa</b> app',
+                    'Go to the <b>Devices</b> page',
+                    'Tap the <b>First light</b> Light Bulb tile to open detailed view',
+                    'Click on the power symbol to turn on the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns on</b>',
+                    'Click on the power symbol again to turn off the Matter Light Bulb',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns off</b>',
+                    'Move the slider to change the brightness of the Matter Light Bulb',
```

Comment:
I guess, here, user should turn the light on once again before changing the brightness. At least this is what the video shows.

Reply:
Done